app/models/wpcc/legal_period_percents.rb

```diff
@@ -0,0 +1,61 @@
+module Wpcc
+  class LegalPeriodPercents
```

Comment:
I have enormous difficult with those names. But even 'PeriodLegalPercents', how to represent the plural (an array of) and singular?